KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/HighlightedText.java

```diff
@@ -359,16 +362,33 @@ public String getText() {
 
             highlightField = LuceneQuery.HIGHLIGHT_FIELD;
             if (isLiteral) {
-                //if the query is literal try to get solr to do the highlighting
-                final String highlightQuery = keywords.stream()
-                        .map(HighlightedText::constructEscapedSolrQuery)
-                        .collect(Collectors.joining(" "));
-
-                q.setQuery(highlightQuery);
-                q.addField(highlightField);
-                q.addFilterQuery(filterQuery);
-                q.addHighlightField(highlightField);
-                q.setHighlightFragsize(0); // don't fragment the highlight, works with original highlighter, or needs "single" list builder with FVH
+                if (2.2 <= indexSchemaVersion) {
```

Comment:
I see references to schema version 2.2 but the schema.xml file has not been updated.

Reply:
Thank you for your review.
I updated `KeywordSearch/release/solr/solr/configsets/AutopsyConfig/conf/schema.xml`. Isn't that sufficient?

Reply:
Yes, but I don't see the change in schema.xml above.

Reply:
The path in my previous comment is wrong. The correct path is `KeywordSearch/solr/solr/configsets/AutopsyConfig/conf/schema.xml
`. 
Diff:
https://github.com/sleuthkit/autopsy/pull/5139/files#diff-3f316c678543ca04318ac83da0a094d8

Reply:
When I click on the link you provided (on this side of the world :-) I see the changes to add the text_ja type and the content_ja and language fields but I do not see any update to the schema version. When I look at the entire file containing your changes I see the following:

<schema name="Autopsy Keyword Search" version="2.1">

What do you see?

Reply:
I noticed that Autopsy is using the version attribute of `<schema>` tag. I will update it later. Thank you for pointing it out.